modules/core/src/test/java/org/apache/ignite/internal/client/thin/BlockingTxOpsTest.java

```diff
@@ -105,17 +107,27 @@ public void testBlockingOps() throws Exception {
                     );
 
                     // Clear operation.
-                    checkOpMultithreaded(client,
-                        () -> cache.put(0, 0),
-                        () -> cache.clear(0),
-                        () -> assertFalse(cache.containsKey(0))
+                    assertThrows(
```

Comment:
Now this checks can be removed. There is no concurrency between put and clear operations in thin client. Exception you check in other test class